app/src/main/res/layout/list_event_item.xml

```diff
@@ -13,109 +13,107 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:card_view="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content">
 
-  <android.support.v7.widget.CardView
+<android.support.v7.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We are using 2 space indentation for XML files. Please update your Android Studio configuration and update the wrong indentations.